src/controllers/users.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { BaseUserDto } from '@/dtos';
+import UserService from '@/services/users.service';
+import { NextFunction, Request, Response } from 'express';
+
+class UsersController {
+  public usersService = new UserService();
+
+  public getUsers = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const userData: BaseUserDto[] = await this.usersService.getUsersFromDB();
+      res.status(200).json({ data: userData });
+    } catch (error) {
+      next(error);
+    }
+  };
+
+  public getSpecificUser = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const ID = Number(req.params.id);
```

Comment:
we don't usually capitalize variable names unless it's for stuff like 
(e.g. BASE_URL, DB_NAME,etc.) at the top file level.
I would keep this as lowercase id.